packages/3d-web-client-core/src/character/CharacterManager.ts

```diff
@@ -55,12 +55,14 @@ export class CharacterManager {
     private readonly sendUpdate: (update: CharacterState) => void,
   ) {
     this.group = new Group();
+    setInterval(() => this.update.bind(this), 3000);
   }
 
   public spawnCharacter(
     characterDescription: CharacterDescription,
     id: number,
     isLocal: boolean = false,
+    spawnPosition: Vector3 = new Vector3(),
   ) {
     this.characterDescription = characterDescription;
     const characterLoadingPromise = new Promise<Character>((resolve) => {
```

Comment:
We could make this synchronous to avoid having `loadingCharacters` and instead manage the mesh loading async (would allow us to show a nameplate where a remote user is before the asset loads).

Reply:
This will not be addressed in my very next PR as it will mainly focus on finishing and polishing the post-processing chain, but the further one will include this refactor. A `TODO` was already inserted at my local branch to ensure it's a priority and on my radar.